__mocks__/fileMock.js

```diff
@@ -0,0 +1 @@
+module.exports = 'test-file-stub'
```

Comment:
I am not too sure about this location. It's really only relevant to the js tests. I would rather just put it in `/volunteer_portal/app/javascript/test/__mocks__/fileMock.js`

Reply:
good point, I'll just have to update the jest config first (by default it points to this directive) :)